Essentials/src/com/earth2me/essentials/commands/Commandrecipe.java

```diff
@@ -26,18 +25,16 @@
     public Commandrecipe() {
         super("recipe");
     }
-    
-    private void disableCommandForVersion1_12() throws Exception {
-        VersionUtil.BukkitVersion version = VersionUtil.getServerBukkitVersion();
-        if (version.isHigherThanOrEqualTo(VersionUtil.v1_12_0_R01)
-            && !ess.getSettings().isForceEnableRecipe()) {
```

Comment:
If we're removing this check entirely, remove `Settings#isForceEnableRecipe` too (it's an undocumented config option I used to confirm whether this bug was fixed in newer versions of Spigot).

Reply:
Will do